[BUG] ESP Outline Mode lost FPS after git-10c0984

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

0.1.5 git-10c0984 (and later)

### Operating System

Windows 10

### Minecraft Version

1.20.4

### Describe the bug

In ESP Outline mode, it causes over half of the FPS loss, and the contour has also become thicker.

### Steps to reproduce

Compare git-c249769 with git-10c0984 (and later)

### Client Log

```shell

---------------------------------------------------------------------------

(git-10c0984)


[09:56:10] [Render thread/INFO]: Backend library: LWJGL version 3.3.2-snapshot
[09:56:10] [Render thread/WARN]: Applying workaround: Prevent the NVIDIA OpenGL driver from using broken optimizations (NVIDIA_THREADED_OPTIMIZATIONS)
[09:56:10] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[09:56:10] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
[09:56:10] [Render thread/INFO]: OpenGL Version: 3.3.0 NVIDIA 537.58
[09:56:11] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[09:56:12] [Render thread/INFO]: Launching LiquidBounce v0.1.5 by CCBlueX
[09:56:12] [Render thread/INFO]: Loaded language en_us
[09:56:12] [Render thread/INFO]: Loaded language de_de
[09:56:12] [Render thread/INFO]: Loaded language ja_jp
[09:56:12] [Render thread/INFO]: Loaded language zh_cn
[09:56:12] [Render thread/INFO]: Loaded language ru_ru
[09:56:12] [Render thread/INFO]: Loaded language en_pt
[09:56:12] [Render thread/INFO]: Loaded language pt_br
[09:56:13] [Render thread/WARN]: Unable to find main.js or main.mjs inside the directory lib.
[09:56:15] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyLogFilter 0.1.js'.
[09:56:15] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyPacketDebugger 0.2.js'.
[09:56:15] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyStrongholdFinder 0.1.js'.
[09:56:15] [Render thread/WARN]: mumyLogFilter is missing fallback description key liquidbounce.module.mumyLogFilter.description
[09:56:15] [Render thread/WARN]: mumyPacketDebugger is missing fallback description key liquidbounce.module.mumyPacketDebugger.description
[09:56:16] [Render thread/WARN]: mumyStrongholdFinder is missing fallback description key liquidbounce.module.mumyStrongholdFinder.description
[09:56:16] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor!
[09:56:16] [Render thread/INFO]: Successfully extracted default theme
[09:56:16] [Render thread/INFO]: Inserted 9 components
[09:56:16] [Render thread/INFO]: Successfully loaded config 'enemies'.
[09:56:16] [Render thread/INFO]: (Chat) 
[09:56:16] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7Auto ConfigAuto Config
[09:56:16] [Render thread/INFO]: Successfully loaded config 'modules'.
[09:56:16] [Render thread/INFO]: Successfully loaded config 'commands'.
[09:56:16] [Render thread/INFO]: Successfully loaded config 'friends'.
[09:56:16] [Render thread/ERROR]: Unable to deserialize value proxy
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:397) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1329) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1271) ~[gson-2.10.1.jar:?]
	at net.ccbluex.liquidbounce.config.Value.deserializeFrom(Value.kt:218) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:273) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:228) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:203) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:201) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:144) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:181) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$startClient(class_310.java:17351) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:165) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:89) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:386) ~[gson-2.10.1.jar:?]
	... 24 more
[09:56:16] [Render thread/INFO]: Successfully loaded config 'proxy'.
[09:56:16] [Render thread/INFO]: Successfully loaded config 'accounts'.
[09:56:16] [Render thread/INFO]: Successfully loaded config 'tabs'.
[09:56:16] [Render thread/INFO]: Successfully loaded config 'lang'.
[09:56:16] [Render thread/INFO]: Successfully loaded config 'storage'.
[09:56:16] [netty-websocket/INFO]: Starting Netty server...
[09:56:16] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c] REGISTERED
[09:56:16] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c] BIND: 0.0.0.0/0.0.0.0:15685
[09:56:16] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] ACTIVE
[09:56:16] [netty-websocket/INFO]: Netty server started on port 15685.
[09:56:17] [Render thread/INFO]: java-cef commit: e91a133673f9524c5f7db9bdbad1aff61b3887d7
[09:56:17] [Render thread/INFO]: Task: Downloading Checksum with progress 0.0 %
[09:56:19] [Render thread/INFO]: Checksum matches: false
[09:56:19] [Render thread/INFO]: Platform directory exists: false
[09:56:19] [mcef-downloader/INFO]: Downloading JCEF...
[09:56:19] [mcef-downloader/INFO]: Task: Downloading JCEF with progress 0.0 %
[09:56:19] [Render thread/INFO]: Successfully loaded client!
...
[09:58:37] [Render thread/ERROR]: ########## GL ERROR ##########
[09:58:37] [Render thread/ERROR]: @ Render
[09:58:37] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[09:58:39] [Render thread/INFO]: Loaded 7 recipes
[09:58:40] [Render thread/INFO]: Loaded 1271 advancements
[09:58:40] [Render thread/INFO]: Successfully saved config 'modules'.
[09:58:40] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@589f45cb, url=http://127.0.0.1:15685/default/#/)
[09:58:40] [Render thread/INFO]: Applied 0 biome modifications to 0 of 64 new biomes in 1.784 ms
[09:58:40] [Server thread/INFO]: Starting integrated minecraft server version 1.20.4
[09:58:40] [Server thread/INFO]: Generating keypair
[09:58:40] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[09:58:40] [Server thread/WARN]: Method overwrite conflict for removeIf in modernfix-common.mixins.json:bugfix.paper_chunk_patches.SortedArraySetMixin from mod modernfix, previously written by me.jellysquid.mods.lithium.mixin.world.chunk_tickets.SortedArraySetMixin. Skipping method.
[09:58:41] [Render thread/INFO]: 准备生成区域中：0%
[09:58:41] [Render thread/INFO]: 准备生成区域中：0%
[09:58:41] [Render thread/INFO]: 准备生成区域中：7%
[09:58:41] [Render thread/INFO]: Time elapsed: 1297 ms
[09:58:42] [Server thread/INFO]: Changing view distance to 16, from 10
[09:58:42] [Server thread/INFO]: Changing simulation distance to 12, from 0
[09:58:42] [Server thread/INFO]: mumy_[local:E:fbacbaae] logged in with entity id 63 at (339.64488213072616, 101.49520087700593, -51.51702850662262)
[09:58:42] [Render thread/INFO]: New minimap session initialized!
[09:58:42] [Server thread/INFO]: mumy_加入了游戏
[09:58:42] [Render thread/INFO]: Successfully saved config 'modules'.
[09:58:42] [Render thread/ERROR]: Exception while executing handler.
java.lang.NullPointerException: null
	at net.ccbluex.liquidbounce.features.module.modules.render.ModuleHoleESP.updateScanRegion(ModuleHoleESP.kt:376) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.modules.render.ModuleHoleESP.enable(ModuleHoleESP.kt:309) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:219) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:216) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:181) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$hookWorldChangeEvent(class_310.java:17496) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_18097(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1481(class_310.java:2257) ~[client-intermediary.jar:?]
	at net.minecraft.class_634.method_11120(class_634.java:414) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11567(class_2678.java:57) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11054(class_2678.java:14) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1283) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
[09:58:42] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@589f45cb, url=http://127.0.0.1:15685/default/#/)
[09:58:42] [Render thread/INFO]: Started 2 worker threads
[09:58:42] [Render thread/INFO]: Minimap updated server level id: -1388266664 for world ResourceKey[minecraft:dimension / minecraft:overworld]
[09:58:42] [Render thread/INFO]: Loaded 165 advancements
[09:58:43] [Render thread/INFO]: [ShulkerBoxTooltip Plugins] Registered 19 color keys for mod shulkerboxtooltip
[09:58:43] [Render thread/WARN]: Shader ui_blur could not find uniform named BlurDir in the specified shader program.
[09:58:43] [Render thread/INFO]: Reloading entity icon resources...
[09:58:43] [Render thread/INFO]: Done!
[09:58:44] [Render thread/WARN]: Time from main menu to in-game was 4.6822395 seconds
[09:58:44] [Render thread/WARN]: Total time to load game and open world was 46.52824 seconds
[09:58:45] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] READ: [id: 0xaa0bc83f, L:/127.0.0.1:15685 - R:/127.0.0.1:13808]
[09:58:45] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] READ COMPLETE
[09:58:45] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] READ: [id: 0x6ae0cc19, L:/127.0.0.1:15685 - R:/127.0.0.1:13809]
[09:58:45] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] READ COMPLETE
[09:59:38] [Server thread/INFO]: Saving and pausing game...
[09:59:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[09:59:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[09:59:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[09:59:38] [Render thread/INFO]: Minimap session finalized.
[09:59:38] [Server thread/INFO]: mumy_ lost connection: 连接中断
[09:59:38] [Server thread/INFO]: mumy_退出了游戏
[09:59:38] [Server thread/INFO]: Stopping singleplayer server as player logged out
[09:59:38] [Server thread/INFO]: Stopping server
[09:59:38] [Server thread/INFO]: Saving players
[09:59:38] [Server thread/INFO]: Saving worlds
[09:59:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[09:59:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[09:59:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[09:59:38] [Server thread/INFO]: ThreadedAnvilChunkStorage (新的世界): All chunks are saved
[09:59:38] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[09:59:38] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[09:59:38] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[09:59:39] [Render thread/INFO]: Successfully saved config 'modules'.
[09:59:39] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@589f45cb, url=http://127.0.0.1:15685/default/#/)
[09:59:39] [Render thread/INFO]: Stopping worker threads
[09:59:44] [Render thread/INFO]: Loaded 7 recipes
[09:59:44] [Render thread/INFO]: Loaded 1271 advancements
[09:59:44] [Render thread/INFO]: Successfully saved config 'modules'.
[09:59:44] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@589f45cb, url=http://127.0.0.1:15685/default/#/)
[09:59:44] [Render thread/INFO]: Applied 0 biome modifications to 0 of 64 new biomes in 149.8 μs
[09:59:44] [Server thread/INFO]: Starting integrated minecraft server version 1.20.4
[09:59:44] [Server thread/INFO]: Generating keypair
[09:59:44] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[09:59:45] [Render thread/INFO]: 准备生成区域中：0%
[09:59:45] [Server thread/INFO]: Changing view distance to 16, from 10
[09:59:45] [Server thread/INFO]: Changing simulation distance to 12, from 0
[09:59:45] [Render thread/INFO]: Time elapsed: 246 ms
[09:59:45] [Server thread/INFO]: mumy_[local:E:bb6fd3dd] logged in with entity id 683 at (9.340915659352165, -57.0, 20.196744305908187)
[09:59:45] [Server thread/INFO]: mumy_加入了游戏
[09:59:45] [Render thread/INFO]: New minimap session initialized!
[09:59:45] [Render thread/INFO]: Successfully saved config 'modules'.
[09:59:45] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@589f45cb, url=http://127.0.0.1:15685/default/#/)
[09:59:45] [Render thread/INFO]: Started 2 worker threads
[09:59:45] [Render thread/INFO]: [ShulkerBoxTooltip] Server integration enabled, attempting handshake...
[09:59:45] [Netty Server IO #3/INFO]: [ShulkerBoxTooltip] literal{mumy_}: protocol version: 2.0
[09:59:45] [Netty Local Client IO #1/INFO]: [ShulkerBoxTooltip] Handshake succeeded
[09:59:45] [Netty Local Client IO #1/INFO]: [ShulkerBoxTooltip] Server protocol version: 2.0
[09:59:45] [Render thread/INFO]: Minimap updated server level id: -1529604272 for world ResourceKey[minecraft:dimension / minecraft:overworld]
[09:59:45] [Render thread/INFO]: Loaded 257 advancements
[09:59:45] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] READ: [id: 0x95db3100, L:/127.0.0.1:15685 - R:/127.0.0.1:13830]
[09:59:45] [nioEventLoopGroup-3-1/INFO]: [id: 0xe33efc5c, L:/[0:0:0:0:0:0:0:0]:15685] READ COMPLETE
[09:59:45] [Render thread/WARN]: Time from main menu to in-game was 1.0345521 seconds
[09:59:45] [Render thread/WARN]: Total time to load game and open world was 42.880554 seconds
[10:02:18] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-12_10.02.17.png
[10:02:22] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-12_10.02.22.png
[10:02:27] [Server thread/INFO]: Saving and pausing game...
[10:02:27] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[10:02:27] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[10:02:27] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[10:02:28] [Render thread/INFO]: Minimap session finalized.
[10:02:28] [Server thread/INFO]: mumy_ lost connection: 连接中断
[10:02:28] [Server thread/INFO]: mumy_退出了游戏
[10:02:28] [Server thread/INFO]: Stopping singleplayer server as player logged out
[10:02:28] [Server thread/INFO]: Stopping server
[10:02:28] [Server thread/INFO]: Saving players
[10:02:28] [Server thread/INFO]: Saving worlds
[10:02:28] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[10:02:28] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[10:02:28] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[10:02:28] [Server thread/INFO]: ThreadedAnvilChunkStorage (新的世界 (2)): All chunks are saved
[10:02:28] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[10:02:28] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[10:02:28] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[10:02:28] [Render thread/INFO]: Successfully saved config 'modules'.
[10:02:28] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@589f45cb, url=http://127.0.0.1:15685/default/#/)
[10:02:28] [Render thread/INFO]: Stopping worker threads
[10:02:31] [Render thread/INFO]: Shutting down client...
[10:02:32] [Render thread/INFO]: Successfully shutdown browser.
[10:02:32] [Render thread/INFO]: Successfully saved config 'enemies'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'modules'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'commands'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'friends'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'proxy'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'accounts'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'tabs'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'lang'.
[10:02:32] [Render thread/INFO]: Successfully saved config 'storage'.
[10:02:32] [Render thread/INFO]: Stopping!
[10:02:32] [Render thread/INFO]: Successfully saved config 'modules'.

---------------------------------------------------------------------------

(git-c249769)


[10:03:33] [Render thread/INFO]: Backend library: LWJGL version 3.3.2-snapshot
[10:03:33] [Render thread/WARN]: Applying workaround: Prevent the NVIDIA OpenGL driver from using broken optimizations (NVIDIA_THREADED_OPTIMIZATIONS)
[10:03:34] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[10:03:34] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2
[10:03:34] [Render thread/INFO]: OpenGL Version: 3.3.0 NVIDIA 537.58
[10:03:34] [Render thread/INFO]: [LambdaBetterGrass] Rebuilding resources...
[10:03:35] [Render thread/INFO]: Launching LiquidBounce v0.1.5 by CCBlueX
[10:03:35] [Render thread/INFO]: Loaded language en_us
[10:03:35] [Render thread/INFO]: Loaded language de_de
[10:03:35] [Render thread/INFO]: Loaded language ja_jp
[10:03:35] [Render thread/INFO]: Loaded language zh_cn
[10:03:35] [Render thread/INFO]: Loaded language ru_ru
[10:03:35] [Render thread/INFO]: Loaded language en_pt
[10:03:35] [Render thread/INFO]: Loaded language pt_br
[10:03:36] [Render thread/WARN]: Unable to find main.js or main.mjs inside the directory lib.
[10:03:39] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyLogFilter 0.1.js'.
[10:03:39] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyPacketDebugger 0.2.js'.
[10:03:39] [Render thread/INFO]: [ScriptAPI] Successfully loaded script 'mumyStrongholdFinder 0.1.js'.
[10:03:39] [Render thread/WARN]: mumyLogFilter is missing fallback description key liquidbounce.module.mumyLogFilter.description
[10:03:39] [Render thread/WARN]: mumyPacketDebugger is missing fallback description key liquidbounce.module.mumyPacketDebugger.description
[10:03:39] [Render thread/WARN]: mumyStrongholdFinder is missing fallback description key liquidbounce.module.mumyStrongholdFinder.description
[10:03:39] [Render thread/INFO]: Successfully extracted default theme
[10:03:39] [Render thread/INFO]: Inserted 9 components
[10:03:39] [Render thread/INFO]: Successfully loaded config 'enemies'.
[10:03:39] [Render thread/INFO]: (Chat) 
[10:03:39] [Render thread/INFO]: (Chat) §f§lLiquid§9§lBounce §8? §7Auto ConfigAuto Config
[10:03:39] [Render thread/ERROR]: Unable to deserialize active choice for Mode
java.lang.IllegalArgumentException: ChoiceConfigurable `Mode` has no option named Glow (available options are Box)
	at net.ccbluex.liquidbounce.config.ChoiceConfigurable.setFromValueName(ConditionalConfigurables.kt:102) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:242) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:228) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:263) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:228) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:203) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:201) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:144) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:180) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$startClient(class_310.java:17351) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
[10:03:40] [Render thread/INFO]: Successfully loaded config 'modules'.
[10:03:40] [Render thread/INFO]: Successfully loaded config 'commands'.
[10:03:40] [Render thread/INFO]: Successfully loaded config 'friends'.
[10:03:40] [Render thread/ERROR]: Unable to deserialize value proxy
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:397) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1329) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1271) ~[gson-2.10.1.jar:?]
	at net.ccbluex.liquidbounce.config.Value.deserializeFrom(Value.kt:218) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:273) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:228) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:203) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:201) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:144) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:180) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$startClient(class_310.java:17351) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:165) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:89) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:386) ~[gson-2.10.1.jar:?]
	... 24 more
[10:03:40] [Render thread/INFO]: Successfully loaded config 'proxy'.
[10:03:40] [Render thread/INFO]: Successfully loaded config 'accounts'.
[10:03:40] [Render thread/INFO]: Successfully loaded config 'tabs'.
[10:03:40] [Render thread/INFO]: Successfully loaded config 'lang'.
[10:03:40] [Render thread/INFO]: Successfully loaded config 'storage'.
[10:03:40] [netty-websocket/INFO]: Starting Netty server...
[10:03:40] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362] REGISTERED
[10:03:40] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362] BIND: 0.0.0.0/0.0.0.0:15000
[10:03:40] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362, L:/[0:0:0:0:0:0:0:0]:15000] ACTIVE
[10:03:40] [netty-websocket/INFO]: Netty server started on port 15000.
[10:03:40] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor!
[10:03:41] [Render thread/INFO]: java-cef commit: e91a133673f9524c5f7db9bdbad1aff61b3887d7
[10:03:41] [Render thread/INFO]: Task: Downloading Checksum with progress 0.0 %
[10:03:43] [Render thread/INFO]: Checksum matches: true
[10:03:43] [Render thread/INFO]: Platform directory exists: true
[10:03:43] [Render thread/INFO]: Initializing CEF on windows_amd64...
[10:03:43] [Render thread/INFO]: [STDOUT]: initialize on Thread[Render thread,5,main] with library path H:\Games\Play2\.minecraft\versions\1.20.4-Fabric 0.15.7\LiquidBounce\mcef\libraries\windows_amd64
[10:03:43] [Render thread/INFO]: Chromium Embedded Framework initialized
[10:03:43] [Render thread/INFO]: Browser is ready.
[10:03:43] [Render thread/INFO]: Successfully loaded client!
...
[10:03:55] [Render thread/ERROR]: ########## GL ERROR ##########
[10:03:55] [Render thread/ERROR]: @ Render
[10:03:55] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[10:03:55] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xd3bca2c6, L:/127.0.0.1:15000 - R:/127.0.0.1:13912]
[10:03:55] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[10:03:57] [Render thread/WARN]: Game took 38.149 seconds to start
[10:04:01] [Render thread/ERROR]: ########## GL ERROR ##########
[10:04:01] [Render thread/ERROR]: @ Render
[10:04:01] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[10:04:05] [Render thread/INFO]: Loaded 7 recipes
[10:04:05] [Render thread/INFO]: Loaded 1271 advancements
[10:04:05] [Render thread/INFO]: Successfully saved config 'modules'.
[10:04:05] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@21d05e35, url=http://127.0.0.1:15000/default/#/)
[10:04:05] [Render thread/INFO]: Applied 0 biome modifications to 0 of 64 new biomes in 1.786 ms
[10:04:05] [Server thread/INFO]: Starting integrated minecraft server version 1.20.4
[10:04:05] [Server thread/INFO]: Generating keypair
[10:04:05] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[10:04:05] [Server thread/WARN]: Method overwrite conflict for removeIf in modernfix-common.mixins.json:bugfix.paper_chunk_patches.SortedArraySetMixin from mod modernfix, previously written by me.jellysquid.mods.lithium.mixin.world.chunk_tickets.SortedArraySetMixin. Skipping method.
[10:04:06] [Render thread/INFO]: 准备生成区域中：0%
[10:04:06] [Render thread/INFO]: 准备生成区域中：0%
[10:04:06] [Render thread/INFO]: Time elapsed: 924 ms
[10:04:06] [Server thread/INFO]: Changing view distance to 16, from 10
[10:04:06] [Server thread/INFO]: Changing simulation distance to 12, from 0
[10:04:07] [Server thread/INFO]: mumy_[local:E:a7fbae46] logged in with entity id 9 at (9.026931148727542, -60.0, 8.294117783700514)
[10:04:07] [Render thread/INFO]: New minimap session initialized!
[10:04:07] [Server thread/INFO]: mumy_加入了游戏
[10:04:07] [Render thread/INFO]: Successfully saved config 'modules'.
[10:04:07] [Render thread/ERROR]: Exception while executing handler.
java.lang.NullPointerException: null
	at net.ccbluex.liquidbounce.features.module.modules.render.ModuleHoleESP.updateScanRegion(ModuleHoleESP.kt:376) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.modules.render.ModuleHoleESP.enable(ModuleHoleESP.kt:309) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:219) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.features.module.Module$onWorldChange$1.invoke(Module.kt:216) ~[liquidbounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:180) ~[liquidbounce.jar:?]
	at net.minecraft.class_310.handler$dcg000$liquidbounce$hookWorldChangeEvent(class_310.java:17496) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_18097(class_310.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1481(class_310.java:2257) ~[client-intermediary.jar:?]
	at net.minecraft.class_634.method_11120(class_634.java:414) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11567(class_2678.java:57) ~[client-intermediary.jar:?]
	at net.minecraft.class_2678.method_11054(class_2678.java:14) ~[client-intermediary.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:115) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1283) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[1.20.4-Fabric%200.15.7.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.7.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.7.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:51) ~[?:?]
[10:04:07] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@21d05e35, url=http://127.0.0.1:15000/default/#/)
[10:04:07] [Render thread/INFO]: Started 2 worker threads
[10:04:07] [Render thread/INFO]: Minimap updated server level id: -1529604272 for world ResourceKey[minecraft:dimension / minecraft:overworld]
[10:04:07] [Render thread/INFO]: Loaded 257 advancements
[10:04:07] [Render thread/INFO]: [ShulkerBoxTooltip Plugins] Registered 19 color keys for mod shulkerboxtooltip
[10:04:08] [Render thread/WARN]: Shader ui_blur could not find uniform named BlurDir in the specified shader program.
[10:04:08] [Render thread/INFO]: Reloading entity icon resources...
[10:04:08] [Render thread/INFO]: Done!
[10:04:09] [Render thread/WARN]: Time from main menu to in-game was 3.7473292 seconds
[10:04:09] [Render thread/WARN]: Total time to load game and open world was 41.896328 seconds
[10:04:09] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362, L:/[0:0:0:0:0:0:0:0]:15000] READ: [id: 0xa624fcc1, L:/127.0.0.1:15000 - R:/127.0.0.1:13918]
[10:04:09] [nioEventLoopGroup-3-1/INFO]: [id: 0x5e5e1362, L:/[0:0:0:0:0:0:0:0]:15000] READ COMPLETE
[10:04:15] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-12_10.04.15.png
[10:04:24] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-12_10.04.24.png
[10:04:34] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-12_10.04.34.png
[10:04:43] [Render thread/INFO]: [CHAT] 已将截图保存为2024-03-12_10.04.43.png
[10:04:50] [Server thread/INFO]: Saving and pausing game...
[10:04:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[10:04:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[10:04:50] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[10:04:52] [Render thread/INFO]: Shutting down client...
[10:04:52] [Render thread/INFO]: Successfully shutdown browser.
[10:04:52] [Render thread/INFO]: Successfully saved config 'enemies'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'modules'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'commands'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'friends'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'proxy'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'accounts'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'tabs'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'lang'.
[10:04:52] [Render thread/INFO]: Successfully saved config 'storage'.
[10:04:52] [Render thread/INFO]: Stopping!
[10:04:52] [Render thread/INFO]: Minimap session finalized.
[10:04:52] [Server thread/INFO]: mumy_ lost connection: 连接中断
[10:04:52] [Server thread/INFO]: mumy_退出了游戏
[10:04:52] [Server thread/INFO]: Stopping singleplayer server as player logged out
[10:04:52] [Server thread/INFO]: Stopping server
[10:04:52] [Server thread/INFO]: Saving players
[10:04:52] [Server thread/INFO]: Saving worlds
[10:04:52] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:overworld
[10:04:53] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_nether
[10:04:53] [Server thread/INFO]: Saving chunks for level 'ServerLevel[新的世界]'/minecraft:the_end
[10:04:53] [Server thread/INFO]: ThreadedAnvilChunkStorage (新的世界 (2)): All chunks are saved
[10:04:53] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[10:04:53] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[10:04:53] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[10:04:53] [Render thread/INFO]: Successfully saved config 'modules'.
[10:04:53] [Render thread/INFO]: Stopping worker threads
```


### Screenshots

## git-10c0984
![2024-03-12_10 02 17](https://github.com/CCBlueX/LiquidBounce/assets/161578211/e68dfc0d-e4d6-4f38-a19c-75d73bd7901a)
![2024-03-12_10 02 22](https://github.com/CCBlueX/LiquidBounce/assets/161578211/fbdc4bec-e882-40cb-8acd-66de3137f6f8)

## git-c249769
![2024-03-12_10 04 34](https://github.com/CCBlueX/LiquidBounce/assets/161578211/323bdc07-4968-448b-8c47-338931625f2e)
![2024-03-12_10 04 43](https://github.com/CCBlueX/LiquidBounce/assets/161578211/52617928-d45e-49bf-b717-6f2cf651201c)

--- a/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/MixinWorldRenderer.java
+++ b/src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/MixinWorldRenderer.java
@@ -76,7 +76,7 @@ public abstract class MixinWorldRenderer {
         try {
             OutlineShader outlineShader = OutlineShader.INSTANCE;
 
-            outlineShader.begin(3.0F);
+            outlineShader.begin(2.0F);
 
             outlineShader.getFramebuffer().beginWrite(false);
 

